db/sql/schema_and_tables.sql

```diff
@@ -0,0 +1,56 @@
+--Schema
+
+CREATE SCHEMA IF NOT EXISTS facilities_lds;
```

Comment:
To be consistent with buildings, there should be 2 schemas and 2 tables.
- `facilities` schema should have a `facilities` table including internal columns
- `facilities_lds` schema should have a `nz_facilities` table with only the columns to be published on the LDS